extensions/wikia/ARecoveryEngine/js/spec/adBlockDetection.spec.js

```diff
@@ -0,0 +1,70 @@
+/*global describe, it, expect, modules, spyOn, beforeEach*/
+describe('ext.wikia.aRecoveryEngine.adBlockDetection', function () {
+	'use strict';
+
+	function noop() {}
+
+	var mocks = {
+			log: noop,
+			context: {
+				opts: {},
+				targeting: {
+					skin: 'oasis'
+				}
+			},
+			adContext: {
+				addCallback: noop,
+				getContext: function () {
+					return mocks.context;
+				}
+			},
+			lazyQueue: {
+				makeQueue: noop
+			},
+			sourcePoint: {
+				isBlocking: noop,
+				isEnabled: noop
+			}
+		};
+
+	mocks.log.levels = {};
+
+	function getModule() {
+		return modules['ext.wikia.aRecoveryEngine.adBlockDetection'](
+			mocks.adContext,
+			mocks.sourcePoint,
+			null,
+			null,
+			mocks.lazyQueue,
+			mocks.log,
+			null
+		);
+	}
+
+	it('isBlocking is true when sourcePointDetection is enabled and SP is blocking', function () {
```

Comment:
Shouldn't "and SP is blocking" be replaced with "SP detects blocking user" or something similar?